src/mscorlib/src/System/Runtime/Intrinsics/X86/Sse.cs

```diff
@@ -185,6 +320,11 @@ public static class Sse
         /// </summary>
         public static Vector128<float> Sqrt(Vector128<float> value) => Sqrt(value);
 
+        /// <summary>
+        /// __m128 _mm_sqrt_ss (__m128 a)
+        /// </summary>
+        public static Vector128<float> SqrtScalar(Vector128<float> value) => SqrtScalar(value);
```

Comment:
The `Sse2` form for `double` takes two arguments, but we only take one here (this is matching the C/C++ intrinsics). Perhaps we should expose both or just the one that takes two arguments?